js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -1793,7 +1793,7 @@ ssoServiceUrl=Single Sign-On service URL
 artifactResolutionServiceUrl=Artifact Resolution service URL
 inputHelperTextAfter=Helper text (under) the input field
 appliedByClients=Applied by the following clients
-createFlowHelp=You can create a top level flow within this from
+createFlowHelp=You can create a top level flow within this from.
```

from or form?

---

It is `form`, I'll correct it in the next commit.

```suggestion
createFlowHelp=You can create a top level flow within this form.
```